client/src/routes/challs.js

```diff
@@ -132,6 +154,22 @@ const Challenges = ({ classes }) => {
         }
       })
     }
+    for (const metatag in tags) {
+      let filterMetatag = false
+      for (const selected of Object.values(tags[metatag])) {
+        if (selected) {
+          filterMetatag = true
+          break
+        }
+      }
+      if (filterMetatag) {
+        const expectedTags = Object.keys(tags[metatag]).filter((tag) => tags[metatag][tag])
```

Comment:
Performance probably doesn't matter here but I would still switch this to a `Set`